RELEASES.md

```diff
@@ -1,5 +1,128 @@
+Version 1.3.0 (September 2015)
+==============================
+
+* ~900 changes, numerous bugfixes
+
+Highlights
+----------
+
+* [The Rustonomicon][nom] is a new book in the official documentation
+  that dives into writing unsafe Rust.
+* The [`Duration`] API, [has been stabilized][ds]. This basic unit of
+  timekeeping is employed by other std APIs, as well as out-of-tree
+  time crates.
+
+Breaking Changes
+----------------
+
+* The `#[prelude_import]`, an internal implementation detail, was
+  accidentally stabilized previously. [It has been put behind the
+  `prelude_import` feature gate][pi]. This change is believed to break
+  no existing code.
+
+Language
+--------
+
+* Semicolons may [now follow types and paths in
+  macros](https://github.com/rust-lang/rust/pull/27000).
+
+Libraries
+---------
+
+* The [`Duration`] API, [has been stabilized][ds], as well as the
+  `std::time` module, which presently contains only `Duration`.
+* `Box<str>` and `Box<[T]>` both implement `Clone`.
+* The owned C string, [`CString`], implements [`Borrow`] and the
+  borrowed C string, [`CStr`], implements [`ToOwned`]. The two of
+  these allow C strings to be borrowed and cloned in generic code.
+* [`CStr`] implements [`Debug`].
+* [`AtomicPtr`] implements [`Debug`].
+* [`Error`] trait objects [can be downcast to their concrete types][e]
+  in many common configurations, using the [`is`], [`downcast`],
+  [`downcast_ref`] and [`downcast_mut`] methods, similarly to the
+  [`Any`] trait.
+* The [`contains`] method for searching strings now [employes the
+  two-way algorithm][search] instead of doing a naive search, for
+  significant speedups.
+* The performance of `PartialEq` for slices is [much faster][ps].
+* The [`Hash`] trait offers the default method, [`hash_slice`], which
+  is overridden and optimized by the implementations for scalars.
+* The [`Hasher`] trait now has a number of specialized `write_*`
+  methods for primitive types, for efficiency.
+* The I/O-specific error type, [`std::io::Error`][ie], gained a set of
+  methods for accessing the 'inner error', if any: [`get_ref`][iegr],
+  [`get_mut`][iegm], [`into_inner`][ieii].
+* [`process::Child`][pc] gained the [`id`] method, which returns a
+  `u32` representing the platform-specific process handle.
```

Comment:
Not a handle! Just a process identifier. There is also an associated handle for each process which is exposed using `AsRawHandle`.